src/utils/get-lybsin-podcasts.js

```diff
@@ -0,0 +1,47 @@
+// Fetches podcast info from https://mongodb.libsyn.com/rss
+
+const RSS_URL = `https://mongodb.libsyn.com/rss`;
+let prodcastList = [];
```

Comment:
```suggestion
const prodcastList = [];
```
nit, but we never re-assign `podcastList` so we can use `const` here. This also can get a bit confusing since there is a `podcastList` in local scope for `getLybsinPodcasts`. Perhaps this variable can just move into `parsePodcasts`?